fem/libceed/ceed.cpp

```diff
@@ -415,6 +415,57 @@ void CeedPAAssemble(const CeedPAOperator& op,
    CeedVectorCreate(ceed, fes.GetNDofs(), &ceedData.v);
 }
 
+void CeedAssembleDiagonalPA(const CeedData ceedDataPtr, Vector &diag)
```

Comment:
Can you invert the order of definition fo `CeedAssembleDiagonalPA` and `CeedAddMultPA`, I think it's more logical, and it's also conforming with the `hpp` file.

Reply:
Done